controllers/device_controller.go

```diff
@@ -232,16 +215,16 @@ func (r *DeviceReconciler) reconcileUpdateDevice(ctx context.Context, d *devicev
 	}
 
 	// 2. reconciling the device properties' value
-	log.V(3).Info("reconciling the device properties")
+	klog.V(3).Info("reconciling the device properties")
 	// property updates are made only when the device is operational and unlocked
 	if newDeviceStatus.OperatingState == devicev1alpha1.Enabled && newDeviceStatus.AdminState == devicev1alpha1.UnLocked {
-		newDeviceStatus, failedPropertyNames = r.reconcileDeviceProperties(d, newDeviceStatus, log)
+		newDeviceStatus, failedPropertyNames = r.reconcileDeviceProperties(d, newDeviceStatus)
 	}
 
 	d.Status = *newDeviceStatus
 
 	// 3. update the device status on OpenYurt
-	log.V(3).Info("update the device status")
+	klog.V(3).Info("update the device status")
```

Comment:
Adding device name into log message may be better.

Reply:
The deviceName has been added.